src/pages/WishlistPage.tsx

```diff
@@ -0,0 +1,69 @@
+
+import './App.css';
```

Comment:
The code was very clean and readable. But there seems to be a small issue with the `WishlistPage.tsx` file where App.css is imported. 
App.css file doesn't exist in our current develop branch. It's causing some errors right now.